pallets/capacity/src/lib.rs

```diff
@@ -773,31 +904,121 @@ impl<T: Config> Pallet<T> {
 		CapacityLedger::<T>::insert(target, capacity_details);
 	}
 
+	fn get_staking_type_and_releasable_percent_in_force(
```

Comment:
This function could use a high level explanation of what it's doing.

Reply:
Might still need it, but it is overhaul'd in https://github.com/frequency-chain/frequency/pull/2439